### DISCUSSION TITLE
How would you go about typesetting something like this?

### DISCUSSION THREAD

**Discussion by A-Walrus** [typst/typst#3944] [Q&A]
![image](https://github.com/typst/typst/assets/58790821/04541b21-9716-4765-a16b-4888f7f23a26)

Where we want the colons (`:`) to be aligned, between items in a bullet list.

I know this could be done with a grid, but I'm wondering if there is a nicer way?

**Comment by istudyatuni** [comment]
You can use `#hide[for]` for remaining lines

**Reply by A-Walrus** [reply]
While that would probably look right, it won't be exact since the width of each of the numbers (or the `n`) is not identical so the colons won't be exactly on top of one another.

**Reply by istudyatuni** [reply]
Try to use grid
